base/module/include/motis/module/module.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <functional>
+#include <map>
```

Comment:
Braucht man hier eigentlich nicht auch noch, wenn es in subc_reg.h schon included wird.